source/commands/danger-runner.ts

```diff
@@ -28,6 +29,9 @@ program
   .description(
     "Handles running the Dangerfile, expects a DSL from STDIN, which should be passed from `danger` or danger run`. You probably don't need to use this command."
   )
+  // Because other calls will trigger this one,
+  // and we don't want to keep a white/blacklist
+  .allowUnknownOption(true)
```

Comment:
can't really write tests for the commands themselves, so I'm letting myself off here